lib/class-wp-theme-json-gutenberg.php

```diff
@@ -790,6 +791,69 @@ public function __construct( $theme_json = array( 'version' => WP_Theme_JSON_Gut
 		}
 	}
 
+	/**
+	 * Unwraps shared block style variations.
+	 *
+	 * @since 6.6.0
+	 *
+	 * {
+	 *   "styles": {
+	 *     "blocks": {
+	 *       "variations": {
+	 *         "blockTypes": [ "core/paragraph", "core/group" ],
+	 *         "color": { "background": "backgroundColor" }
+	 *       }
+	 *     }
+	 *   }
+	 * }
```

Comment:
I noticed it doesn't seem to match what the function does (there's a `blocks` key and the variation name is missing). Here's an attempt at a fix:
```suggestion
	 * {
	 *   "styles": {
	 *     "variations": {
	 *       "light": {
	 *         "blockTypes": [ "core/paragraph", "core/group" ],
	 *         "color": { "background": "backgroundColor" }
	 *       }
	 *     }
	 *   }
	 * }
```

Reply:
Ah thanks, I'd only just spotted that copy and paste from https://github.com/WordPress/wordpress-develop/pull/6868. I've tweaked it and can double check its accuracy as this settles.